librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -3,6 +3,251 @@
 ---
 name: cmd_debug
 commands:
+  - name: db
+    summary: Breakpoints commands
+    subcommands:
+      - name: db
+        summary: List breakpoints / Add breakpoint
+        cname: cmd_debug_db
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: db*
+        summary: List breakpoints in r commands
+        cname: cmd_debug_list_bp_r
+        args: []
+      - name: dbH
+        summary: Add hardware breakpoint
+        cname: cmd_debug_add_hw_bp
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: db-
+        summary: Remove breakpoint
+        cname: cmd_debug_remove_bp
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: db-*
+        summary: Remove all breakpoints
+        cname: cmd_debug_remove_all_bp
+        args: []
+      - name: db.
+        summary: Show breakpoint info at current offset
+        cname: cmd_debug_show_cur_bp
+        args: []
+      - name: dbj
+        summary: List breakpoints in JSON format
+        cname: cmd_debug_list_bp_json
+        args: []
+      - name: dbc
+        summary: Run a command when the breakpoint is hit
+        cname: cmd_debug_command_bp
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+          - name: cmd
+            type: RZ_CMD_ARG_TYPE_CMD
+      - name: dbC
+        summary: Run command and continue until <cmd> returns zero
```

Comment:
I'm not sure how to use this command in practice... I think all these `dbc`/`dbC` are a bit broken... Anyway, I feel like this should be `Stop on breakpoint at <addr> only when <cmd> returns 0`. Though i'm not 100% sure... @yossizap ?

Reply:
Yes. `dbC` basically does what you said, from what I remember. I also felt the description is a bit misleading.

Reply:
>Stop on breakpoint at <addr> only when <cmd> returns 0

Yep that's the intention.

Since this command isn't very clear I would also add an example with a basic condition like `?v $.rax-0x0`

Reply:
woah.. what is `$.rax`?

Reply:
I am not sure but it doesn't seem to make a difference, ignore that part

Reply:
> Since this command isn't very clear I would also add an example with a basic condition like `?v rax-0x0`

Where should I add the example?

Reply:
You can add an examples field under `details` that will show up in `dbC??`, check `cmd_zign.yml` for a reference implementation.